src/models/MigrationDataElements.ts

```diff
@@ -17,7 +17,7 @@ const fields: DefineModelAttributes<object> = {
   dataElementCode: STRING,
   organizationUnitCode: STRING,
   isProcessed: BOOLEAN,
-  migratedAt: STRING,
+  migratedAt: Date,
```

Comment:
should be consistent with Sequilize Types

Reply:
Changed date to use sequelize DATE type